io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties

```diff
@@ -4,6 +4,7 @@ io.remove=Operation Removes subsystem
 io.worker=Defines workers
 io.worker.add=Adds new worker
 io.worker.remove=Removes worker
+io.worker.task-core-threads=Number of task threads with which worker starts with.
```

Comment:
Text should be same as in wildfly-io_3_0.xsd - 
https://github.com/wildfly/wildfly-core/pull/3107/files#diff-4f40ca798a3d232a7d9cc3fb027ebdd5R78

```
                        Specify the starting number of threads for the worker task thread pool.
                        If not set, it defaults to 2.
```